### ISSUE TITLE
Using the `.summary()` method when fitting with weights produces an error.

### ISSUE THREAD

**Issue by MosesStewart** [s3alfisc/pyfixest#350]
I included sample code below:

```
import pyfixest as pf

data = pf.get_data()
res = pf.feols("Y ~ X1 | f1 + f2", data=data, weights = 'weights')
res.summary()
```

Calling `res.summary()` produces a `TypeError` on `PyFixest` 0.17.1 and 0.17.4

**Comment by s3alfisc** [comment]
Thanks for reporting! I can fix this later tonight =)

**Comment by s3alfisc** [comment]
Ah yes, it is related to #348 - `.summary()` tries to print an R2 value, which is not yet computed for WLS.

**Comment by s3alfisc** [comment]
Hi, I just merged version `0.17.3` to master, you can install it by running 

```
pip install git+https://github.com/s3alfisc/pyfixest.git
```

Thanks again for reporting!
